src/json.hpp.re2c

```diff
@@ -6168,6 +6168,21 @@ class basic_json
               thousands_sep(!loc->thousands_sep ? '\0' : loc->thousands_sep[0]),
               decimal_point(!loc->decimal_point ? '\0' : loc->decimal_point[0])
         {}
+        serializer(const serializer& cpy)
```

Comment:
I added my proposed two lines to `json.hpp` and the `-Weffc++` warnings disappeared. I think there is no need to change other code